.gitignore

```diff
@@ -1,5 +1,4 @@
 target
 **/*.rs.bk
```

Comment:
I don't think this meaningful to still have https://github.com/rust-lang/rustfmt/issues/1114
```suggestion
```

Reply:
Done.